Shopipy.ServiceDefaults/Extensions.cs

```diff
@@ -2,12 +2,13 @@
 using Microsoft.AspNetCore.Diagnostics.HealthChecks;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
 using OpenTelemetry;
 using OpenTelemetry.Metrics;
 using OpenTelemetry.Trace;
 
-namespace Microsoft.Extensions.Hosting;
+namespace Shopipy.ServiceDefaults;
```

Comment:
this probably is the line that messed up some imports in Web project. Do not pollute the diff!